src/Microsoft.AspNet.Mvc.Razor/RazorTextWriter.cs

```diff
@@ -54,7 +59,7 @@ public override Encoding Encoding
         private TextWriter TargetWriter { get; set; }
 
         [RazorInject]
```

Comment:
Shouldn't need this now that value is constructor injected. BTW I'm curious why you made that change?